docs/integration-guides/next-auth.md

```diff
@@ -281,13 +391,15 @@ const BIBindPasskey = () => {
 export default BIBindPasskey;
 ```
 
-**What's happening here?**  
+**What's happening here?**
 
-1. The `useEffect` is only called once on page load. In this function, we initialize the Beyond Identity SDK and use `embedded.isBindPasskeyUrl` to check if the current page that was redirected to is in fact a valid `bind` URL.
+1. After tapping the email sent to the user's device, the user is directed to your application's `/bind` route appending to the `invoke_url` configured in the application config.
 
-2. If the URL is valid, we pull the URL using `window.location.href` and pass that directly into `embedded.bindPasskey` to complete the binding process.
+2. The `useEffect` is only called once on page load. In this function, we initialize the Beyond Identity SDK and use `embedded.isBindPasskeyUrl` to check if the current page that was redirected to is in fact a valid `bind` URL.
 
-3. Finally, the response of `embedded.bindPasskey` contains a `passkey` object, which represents the passkey bound to the device.
+3. If the URL is valid, we pull the URL using `window.location.href` and pass that directly into `embedded.bindPasskey` to complete the binding process.
```

Comment:
Is there anything actionable in this sentence? If not, I would indent it under the first step.

Also, be careful with the use 'we'. If the user is doing something in this step, say 'you' and put the verb (action) at the beginning, so when they scan the content they'll know what this step does quickly.